mtp_noms_ops/templates/security/prisoners-detail.html

```diff
@@ -2,20 +2,32 @@
 {% load i18n %}
 {% load mtp_common %}
 {% load security %}
+{% load static %}
 
 {% block inner_content %}
 
   <header>
     {% language_switch %}
-    {% include 'security/includes/save-search.html' %}
     <h1 class="heading-xlarge">
-      {{ view.title }}
       {% if prisoner %}
+        {% if nomis_api_available %}
+          <div class="mtp-prisoner-image">
+            {% if photo %}
+              <img src="{{ photo }}" alt="{% trans 'Photo of the prisoner' %}"/>
```

Comment:
should be `data:image/jpeg;base64,{{ photo }}` i think. but also depends on how NOMIS encodes the image – sometimes base64 encoded files include line breaks that we’d have to strip out.